tests/scripts/analyze_outcomes.py

```diff
@@ -460,6 +460,7 @@ class DriverVSReference_ecc_no_bignum(outcome_analysis.DriverVSReference):
         ],
         'test_suite_asn1parse': [
             'INTEGER too large for mpi',
+            re.compile(r'Parse integer raw,.*'),
```

Comment:
Come to think of it, given that the new function doesn't involve the bignum type and doesn't call bignum function in its implementation, should it still be guarded by `MBEDTLS_BIGNUM_C`?

If not then we don't need an exception here.

Reply:
The testcase that parses the number with `mbedtls_mpi_core_read_be()` requires bignum. There is another testcase in the PR that only tests the function itself which doesn't require an exception like this and is not guarded by `MBEDTLS_BIGNUM_C`.

Reply:
[Discussed separately] I will close this and move the relevant testcases to `test_suite_mpi`, which will prevent them from needing this exception.